doc/README.md

```diff
@@ -0,0 +1,421 @@
+# Seedvault App Backup Repository Documentation
+
+This document is a technical description of how Seedvault backs up apps
+installed on an Android device.
+
+It is inspired by the design of the [Restic backup program](https://restic.net/).
+Portions of this document are blatant copies of
+[its documentation](https://restic.readthedocs.io/en/latest/100_references.html).
+However, due to the different nature of Android backups,
+the design [was heavily simplified](#differences-to-restic).
+
+Note that the backup of files is described [in a different document](../storage/doc/design.md).
+
+## Terminology
```

Comment:
[Clarity] There are parts of the Terminology section that seem to shy away from using the word "file" when referring to backup storage. Is this intentional? Is it meant to be more generic so as to potentially apply to e.g. block storage backends as well?

If so, I understand, but if that's the case, the document has numerous other references to "files" that should possibly be altered to avoid confusion.

I've added comments in a few other, specific places related to this.

Reply:
Right. They are most likely files and in all current backends, they are files. However, they could be something else in the backend.